src/composed/cleartext.rs

```diff
@@ -295,7 +299,7 @@ fn validate_headers(headers: Headers) -> Result<Vec<HashAlgorithm>> {
 ///
 /// This implementation is implicitly agnostic between "\n" and "\r\n" line endings.
 ///
-/// Ref https://www.ietf.org/archive/id/draft-ietf-openpgp-crypto-refresh-13.html#name-dash-escaped-text
+/// Ref https://www.rfc-editor.org/rfc/rfc9580.html#name-dash-escaped-text
```

Comment:
why the change to the URLs?

Reply:
RFC 9580 was finally released a few days ago. I think we should point to the finalized RFC URLs, not the (pre-publication) draft.